src/utils.rs

```diff
@@ -32,3 +32,10 @@ pub(crate) unsafe fn try_demangle_from_ffi(c_str: *const std::ffi::c_char) -> Op
         })
     })
 }
+
+pub(crate) fn uuid_from_i8_slice(bytes: &[i8; 16]) -> uuid::Uuid {
+    // Safety: i8 and u8 are both single-byte types.
+    let u8_bytes = unsafe { &*(bytes as *const [i8; 16] as *const [u8; 16]) };
+    // Safety: unwrap because this is not a variable length slice
+    uuid::Uuid::from_slice(u8_bytes).unwrap()
```

Comment:
https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.from_bytes:
```suggestion
    uuid::Uuid::from_bytes(u8_bytes)
```